decode.go

```diff
@@ -1,31 +1,32 @@
 package configor
```

Comment:
Have you considered using upstream https://github.com/jinzhu/configor which is already upgraded to yaml v3?

(I'm okay with merging and not using upstream, just a question)

Reply:
I'm thinking about that as well, I was just looking at this diff and can't figure out what exactly the difference in behavior this cause, the code seems to do more than the commit message says.

84c96aa9c620209bd4f929a5b3fc2549c3f65bc6

I added the test to upstream it passed so I think it should suffice for our case.

Reply:
Not sure anymore why I used this fork. I think it was mostly because of some println stuff. Using jinzhu/configor directly seems to work fine with gotify/server, maybe we use upstream directly instead of the fork?